include/LightGBM/R_object_helper.h

```diff
@@ -39,9 +39,9 @@
 
   // 64bit pointer
   #if INTPTR_MAX == INT64_MAX
-    typedef int64_t R_xlen_t;
+    typedef int64_t xlen_t;
```

Comment:
I had to rename this type to avoid conflicts with  `R_xlen_t` defined in R's headers.